main.py

```diff
@@ -0,0 +1,49 @@
+import requests
+from dotenv import dotenv_values
```

Comment:
I cant run this script because of this dependency, which dependencies this project has?
Create a `requirements.txt` file with the correct dependencies to run this script.